### ISSUE TITLE
aacar 13 axon terminal issue

### ISSUE THREAD

**Issue by tgbugs** [open-physiology/apinatomy-models#13]
axon-bags should be axon-tubes, axons missing real termination location

this affects NPO, ApiNATOMY, and SVG.

**Comment by tgbugs** [comment]
NPO issue is that dorsal funiculus of spinal cord is too granular.

**Comment by tgbugs** [comment]
The svg has been added and updated. @smtifahim see if this clarifies the issue you were having. @bdebono the only notation I added was to convert the marker at the end of the curves for dendrites to an open box, the browser renders it as filled, but I think that is a bug/inconsistency in the web svg renderer.
https://raw.githubusercontent.com/open-physiology/apinatomy-models/master/models/ard-arm-cardiac/docs/ard-arm-cardiac.svg
